github/repos_contents.go

```diff
@@ -72,7 +73,8 @@ func (r *RepositoryContent) Decode() ([]byte, error) {
 	return o, nil
 }
 
-// GetReadme gets the Readme file for the repository.
+// GetReadme gets the Readme file for the repository, which is stored as base64.
```

Comment:
Rather than all the docs various, would it not be sufficient to just document the Content field of RepositoryContent to say that it may be encoded, and if so, then use `Decode` to decode it?

I don't really want to document that Readme is always stored as base64 because I don't actually know that that's true.  Otherwise, why would GitHub bother returning an "encoding" value?